charts/stable/navidrome/values.yaml

```diff
@@ -21,6 +21,9 @@ persistence:
   data:
     enabled: true
     mountPath: "/data"
+  music:
+    enabled: true
+    mountPath: "/music"
```

Comment:
I'd say it's better to mark is as a major version bump here, as this will go back to pvc on update.
Well data won't be lost, but it will require manual config to change it bck to hostpath or whatever and remove additional storage
@Ornias1993 What do you think?